src/overlays/actors/ovl_En_Test7/z_en_test7.c

```diff
@@ -432,184 +419,188 @@ void EnTest7_Destroy(Actor* thisx, PlayState* play) {
     LightContext_RemoveLight(play, &play->lightCtx, this->lightNode);
 }
 
-void func_80AF19A8(EnTest7* this, PlayState* play) {
+void EnTest7_StartWarpCs(EnTest7* this, PlayState* play) {
     if (!CutsceneManager_IsNext(play->playerCsIds[PLAYER_CS_ID_SONG_WARP])) {
         CutsceneManager_Queue(play->playerCsIds[PLAYER_CS_ID_SONG_WARP]);
     } else {
         CutsceneManager_Start(play->playerCsIds[PLAYER_CS_ID_SONG_WARP], NULL);
-        func_80AF082C(this, func_80AF1A2C);
+        EnTest7_SetupAction(this, EnTest7_WarpCsPart1);
         play->unk_18844 = true;
     }
 }
 
-void func_80AF1A2C(EnTest7* this, PlayState* play) {
+void EnTest7_WarpCsPart1(EnTest7* this, PlayState* play) {
     Color_RGB8 fogColor = { 64, 0, 0 };
     Color_RGB8 ambientColor = { 220, 220, 255 };
-    f32 envLerp = this->unk_1E54 / 10.0f;
+    f32 envLerp = this->timer / (f32)(10 - 0);
 
     Environment_LerpAmbientColor(play, &ambientColor, envLerp);
     Environment_LerpFogColor(play, &fogColor, envLerp);
     Environment_LerpFog(play, 2000, 4000, envLerp);
 
-    if (this->unk_1E54 >= 10) {
+    if (this->timer >= 10) {
         Camera* subCam =
             Play_GetCamera(play, CutsceneManager_GetCurrentSubCamId(play->playerCsIds[PLAYER_CS_ID_SONG_WARP]));
 
         this->subCamEye = subCam->eye;
         this->subCamAt = subCam->at;
         this->subCamFov = subCam->fov;
 
-        func_80AF082C(this, func_80AF1CA0);
-        this->unk_144 |= 0x20;
+        EnTest7_SetupAction(this, EnTest7_WarpCsPart2);
+        this->flags |= OWL_WARP_FLAGS_20;
         Audio_PlaySfx_AtPos(&this->actor.projectedPos, NA_SE_PL_WARP_WING_OPEN);
         Play_EnableMotionBlur(120);
     }
 }
 
 void func_80AF1B68(EnTest7* this, PlayState* play) {
```

Comment:
Hmmm, for naming this one, it seems like it's big thing is scaling up the wind capsule and (eventually) hiding the wings. Maybe just `EnTest7_ScaleUpWindCapsule`? Or `EnTest7_GrowWindCapsule`? Something like that.

Reply:
It does seem like this is the main function to grow the wind capsule, but since it does other things like spinning it, changing flags, and setting up another action func, what would you think of:
`EnTest7_UpdateGrowingWindCapsule`

Reply:
Yeah, I think that's fine.